hs_core/robots.py

```diff
@@ -0,0 +1,45 @@
+import re
+
+import robot_detection
+from django.conf import settings
+from django.http import HttpResponseForbidden
+
+
+# note: robot_detection needs to be updated periodically (possible inside the Dockerfile)
+# (wget http://www.robotstxt.org/db/all.txt, python robot_detection.py all.txt)
+
+class CrawlerBlocker:
+    def __init__(self):
+        self.whitelist = getattr(settings, "BOT_WHITELIST", -1)
```

Comment:
Using an empty list rather than -1 for a default value here might make more sense.